### PR TITLE
Update README to the new format for service catalog

### PR DESCRIPTION
**Pull request by yorinasub17**
This PR updates the `README` for `helm-kubernetes-services` as per [the Service Catalog README RFC](https://github.com/gruntwork-io/mission-control/blob/master/rfc/service-catalog-readme.md).

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+If you just want to try this repo out for experimenting and learning, check out the following resources:
+
+* link:/examples[examples folder]: The `examples` folder contains sample code optimized for learning, experimenting, and testing (but not production usage).
+
+=== Production deployment
+
+If you want to deploy this repo in production, check out the following resources:
+
+* https://github.com/gruntwork-io/infrastructure-modules-multi-account-acme/tree/master/services/k8s-service[k8s-service in the Acme example Reference Architecture]: Production-ready sample code from the Acme Reference Architecture examples.
```
Comment (yorinasub17) [inline README.adoc]: @brikis98 How should I indicate that this is a subscriber-only link?

**Comment by yorinasub17** [comment]
I just realized this has been hanging for quite some time! I think I still need an answer to the question above. Maybe @eak12913 can pitch in?

**Comment by eak12913** [comment]
@yorinasub17 internally we're using: 

```
"licenseType": "subscriber"
```
in the datastructure currently. The current values are:

```
licenseType: "subscriber" | "open-source" | "unimplemented";
```

This may be a little bit moot because we're not currently even crawling these frontmatters.

**Comment by yorinasub17** [comment]
Ah let me clarify the question a bit since that doesn't quite answer it:

This repo is currently open source, but the README will now include a link to a repo that is for Gruntwork subscribers under the "Production" list. Should this link to the github page, or the service catalog so that non-subscribers can view something? Or is that automatically handled when the asciidoc is pulled in?

**Comment by yorinasub17** [comment]
Going to merge this in now.

**Pull request merged**